miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -20,6 +20,12 @@ abstract class MiniAppMessageBridge {
         callback: (isGranted: Boolean) -> Unit
     )
 
+    /** Post custom permissions request from external. **/
+    abstract fun requestCustomPermissions(
+        permissions: List<MiniAppCustomPermissionType>,
+        callback: (grantResult: String) -> Unit
```

Comment:
I think we want to receive a list of which permissions were granted and which were denied, not just a single result for all of them. So basically, this should return something like `List<Pair<MiniAppCustomPermissionType, MiniAppCustomPermissionResult>`.

Reply:
I think, this is okay now, we will return callback as String to MiniApp, it will be a json-like string. 
So, `callback: (grantResult: String) -> Unit` is okay.  MiniApp will parse the the json string. 

@ts-leojoseph-r What do you think for iOS?

Reply:
Sorry, I just noticed this comment. @khairul-alam-licon in iOS we have something like `MiniAppCustomPermissionGrantedStatus`